README.md

```diff
@@ -18,75 +17,95 @@ npm start
 
 Then open [`localhost:3001`](localhost:3001) in a browser.
 
+For more details see the source code of [examples](https://github.com/and-who/react-p5-wrapper/tree/master/example/src).
 
 ## Installation
 
-The easiest way to use react-p5-wrapper is to install it from NPM and include it in your own React build process (using [Browserify](http://browserify.org), [Webpack](http://webpack.github.io/), etc).
+The easiest way to use `react-p5-wrapper` is to install it from NPM and include it in your own React build process (using [Browserify](http://browserify.org), [Webpack](http://webpack.github.io/), etc).
```

Comment:
I think the part about build processes isn't necessary, the library works with a standard install and a bundler isn't necessarily required.

Reply:
Agreed, I simplified this section. Thanks for the feedback 💪